lms/djangoapps/instructor/views/api.py

```diff
@@ -2295,10 +2296,9 @@ def list_report_downloads(_request, course_id):
     """
     course_id = SlashSeparatedCourseKey.from_deprecated_string(course_id)
     report_store = ReportStore.from_config(config_name='GRADES_DOWNLOAD')
-
     response_payload = {
         'downloads': [
-            dict(name=name, url=url, link='<a href="{}">{}</a>'.format(url, name))
+            dict(name=name, url=url, link=HTML('<a href="{}">{}</a>').format(url, name))
```

Comment:
@mushtaqak: This fix generally looks fine.  Unfortunately, the test failure needs additional digging to determine what is actually going wrong.  Let me know when you hone in on why this is breaking.